src/routes/DigitalAssets/routes/Grid/sagas/digitalAssetsGrid.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import { select, put, takeEvery } from 'redux-saga/effects'
+import { selectDigitalAssetsGridFilters } from 'store/stateSelectors'
+
+import {
+  setSortByName,
+  setSortByBalance,
+  SORT_BY_NAME_CLICK,
+  SORT_BY_BALANCE_CLICK,
+} from '../modules/digitalAssetsGrid'
+
+function toggleSortDirection(currentDirection: SortDirection): SortDirection {
+  return (currentDirection === 'asc') ? 'desc' : 'asc'
+}
+
+function* sortByNameClick(): Saga<void> {
```

Comment:
we have to move all logic for sorting here from containers.
Also we can reduce actions count, if we add one method for setting of sorted items.
So we should implement just two actions for setting of sorting options (it must auto toggle) and one for setting of sorted items. And just one saga